packages/front-end/services/track.ts

```diff
@@ -199,22 +205,20 @@ export default function track(
 
   void dataWareHouseTrack({
     event_name: event,
-    properties_json: JSON.stringify(trackProps),
+    properties_json: JSON.stringify(props),
     device_id: getOrGenerateDeviceId(),
     page_id: getOrGeneratePageId(),
     session_id: getOrGenerateSessionId(),
     sdk_language: "react",
-    // TODO: programmatically get sdk version. Importing from _app breaks tests
-    sdk_version: "1.2.0",
+    sdk_version: growthbook.version,
     url: trackProps.url,
     user_id: id,
-    user_attributes_json: "{}",
+    user_attributes_json: JSON.stringify({
+      ...growthbook.getAttributes(),
+      cloudOrgId: isCloud() ? org : "",
```

Comment:
Why does this go in user_attributes_json when it isn't an attribute?

Reply:
I know for debugging purposes, it's nice to have the actual org id on Cloud instead of just a hashed version.  I want us to continue using the hashed id for targeting, so was a little hesitant to add this as an attribute.